paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -374,6 +374,15 @@
     data_type : fpn_rois
   optional : rois_num, multi_level_rois_num
 
+- op : distributed_lookup_table
+  args : (Tensor[] ids, Tensor w, int table_id = 0, bool is_distributed = false, str lookup_table_version = "lookup_table", int64_t padding_idx = -1, int dtype = 5, bool is_test = false)
+  output : Tensor[](outputs){ids.size()}
+  infer_meta :
+    func : DistributeLookupTableInferMeta
+  kernel :
+    func : distributed_lookup_table
+    data_type : ids
```

```suggestion
    data_type : dtype
```
这里需要和DistributedLookupTableOp::GetExpectedKernelType 保持一致

---

谢谢，已修改~